tests/transport/compliance.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import flask
+import pytest
+from pytest_localserver.http import WSGIServer
+from six.moves import http_client
+
+from google.auth import exceptions
+
+# .invalid will never resolve, see https://tools.ietf.org/html/rfc2606
+NXDOMAIN = 'test.invalid'
+
+
+class RequestResponseTests(object):
+
+    @pytest.fixture
+    def server(self):
+        """Provides a test HTTP server that is automatically created before
+        a test and destroyed at the end. The server is serving a test
+        application that can be used to verify requests."""
+        # pylint: disable=unused-variable
+        # (pylint thinks the flask routes are unusued.)
+        app = flask.Flask(__name__)
+        app.debug = True
```

Comment:
It's not `app` that's being marked as unused, it's `index` and `server_error`.

I don't `enable=`. Pylint has block scoping on options and I see no reason to clutter up things with enables at the end of the block.